app/__init__.py

```diff
@@ -25,6 +26,8 @@ def load_config(application):
     application.config['API_KEY'] = os.environ['TEMPLATE_PREVIEW_API_KEY']
     application.config['NOTIFY_ENVIRONMENT'] = os.environ['NOTIFY_ENVIRONMENT']
     application.config['NOTIFY_APP_NAME'] = 'template-preview'
+    application.config['DANGEROUS_SALT'] = os.environ['DANGEROUS_SALT']
+    application.config['SECRET_KEY'] = os.environ['SECRET_KEY']
```

Comment:
We don't need these new environment variables for the Flask app, only the Celery app, so this could be changed to `os.environ.get('SECRET_KEY')` instead so that we can leave the variables out of the Flask manifest. Doing things this way seemed slightly safer since it ensures that the variables must always be set for the Celery app.

Reply:
I think I'm okay with leaving this as is. If someone decides to use the encryption class, at least they are using the right keys.